PovertyAction/ipa-data-tech-handbook
# removing test changes

# Pull Request Summary 🚀

## What does this PR do? 📝



Removing file changes that we made during a testing session on 2024-07-11.

## Why is this change needed? 🤔



Clean up and remove the test page so that it isn't visible on the public website.

## How was this implemented? 🛠️


Removed the `docs/guides/mdtestguide.md` file and deleted that page from `_quarto.yml`

## How to test or reproduce ? 🧪



I ran `quarto preview` and `quarto render` to confirm the test page was deleted and content was update.

## Screenshots (if applicable) 📷



## Checklist ✅

- [X] I have limit this PR to less than 1000 lines of code change (if not, explain why)
- [ ] I have updated/added tests to cover my changes (if applicable)
- [ ] I have updated/added requirements to cover my changes (if applicable)
- [X] I have reviewed and resolved any merge conflict

## Reviewer Emoji Legend

|     |         `:code:`          | Meaning                                                                                                                                                                                                                                         |
| :-: | :-----------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 😃👍💯 | `:smiley:` `:+1:` `:100:` | I like this... <br /><br /> ...and I want the author to know it! This is a way to highlight positive parts of a code review.                                                                                                                    |
| ⭐⭐⭐ |  `:star: :star: :star:`   | Important to fix before PR can be approved... <br /><br /> And I am providing reasons why it needs to be addressed as well as suggested improvements.                                                                                           |
| ⭐⭐  |      `:star: :star:`      | Important to fix but non-blocking for PR approval... <br /><br /> And I am providing suggestions where it could be improved either in this PR or later.                                                                                         |
|  ⭐  |         `:star:`          | Give this some thought but non-blocking for PR approval... <br /><br /> ...and consider this a suggestion, not a requirement.                                                                                                                   |
|  ❓  |       `:question:`        | I have a question. <br /><br /> This should be a fully formed question with sufficient information and context that requires a response.                                                                                                        |
|  📝  |         `:memo:`          | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                                                                                 |
|  ⛏  |         `:pick:`          | This is a nitpick. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
| ♻️  |        `:recycle:`        | Suggestion for refactoring. <br /><br /> Should include enough context to be actionable and not be considered a nitpick.                                                                                                                        |
